src/libpgmoneta/restore.c

```diff
@@ -53,10 +53,11 @@
 #define RESTORE_OK            0
 #define RESTORE_MISSING_LABEL 1
 #define RESTORE_NO_DISK_SPACE 2
+#define RESTORE_TYPE_UNKNOWN  3
 #define INCREMENTAL_MAGIC 0xd3ae1f0d
 #define INCREMENTAL_PREFIX_LENGTH (sizeof(INCREMENTAL_PREFIX) - 1)
 #define MANIFEST_FILES "Files"
-#define MAX_PATH_INCREMENTAL (MAX_PATH * 2)
+#define MAX_PATH_CONCACT (MAX_PATH * 2)
```

Comment:
What does name mean ?

Reply:
Path that is paths concatenated together... I just realized I misspelled it, and it's a stupid macro with a silly name only to accommodate snprintf.